src/dom-to-image-more.js

```diff
@@ -558,6 +563,115 @@
             return node;
         });
     }
+
+    function compressSvg(clone) {
+        const sandboxWindow = ensureSandboxWindow();
+        sandboxWindow.document.body.appendChild(clone);
+    
+        // Generate ascending DOM tree for reverse level order traversal.
+        // CSS inheritance is computed downward (preorder traversal) and is additive-cumulative.
+        // The filter op is subtractive and goes upward (to only splice out inheritable style declarations).
+        const walker = document.createTreeWalker(clone, NodeFilter.SHOW_ELEMENT);
```

Comment:
Is createTreeWalker something broadly available? First I've seen it (but that's me)

Reply:
Yep, the first DOM level 2 APIs are older than ES5. They were old when the awful Mutation Events API was new :)

Reply:
Perfect!